### DISCUSSION TITLE
Movelist sort present in Documentation but not in Code.

### DISCUSSION THREAD

**Discussion by apreedee** [Disservin/chess-library#55] [Q&A] [answered]
Hi, I noticed that a `movelist.sort` function for the `movelist` is mentioned in the project documentation. But there was none when I tried to find the code behind it. Could anyone clarify what the problem is? If it is an issue on my end, I am sorry for wasting your time.

**Comment by Disservin** [comment, marked as answer]
Ah I've removed it from the Movelist class because it might lead to different sorting for different environments and a self-implemented sort for chess engines is way faster. I forgot to remove it from the documentation.

**Reply by apreedee** [reply]
Thanks for the reply. I love the project and it has helped me a lot, thanks. Keep up the good work!

**Reply by Disservin** [reply]
Cool happy to hear that!
